src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../types';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getPosts(userId: string): Promise<Post[]> {
+  const response = await fetch(`${BASE_URL}/posts?userId=${userId}`);
```

Comment:
We should handle erros in the fetch request, and return a message to the client if there is an error.